src/views/ProfileEditView/profileEditView.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ProfileEditComponent from '../../components/Profile/ProfileEdit';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+function setup() {
+  const profile = {};
+  const handleSubmit = () => ({});
+  const onChange = () => ({});
+
+  const wrapper = shallow(
+    <ProfileEditComponent handleSubmit={handleSubmit} profile={profile} onChange={onChange} />,
+);
+
+  return {
```

Comment:
Why are you returning the wrapper?

Reply:
This wrapper shrouds the "ProfileEditComponent"
It is a more elaborate way of implementing the wrapper function.
It calls
<img width="480" alt="screenshot 2019-02-28 at 11 52 15" src="https://user-images.githubusercontent.com/39020475/53553716-553c9680-3b4f-11e9-8947-a88922aa04cd.png">

And tests that a button exists within the ProfileEditComponent.

Regards
Steven Ennis